packages/config/config/devices/0x0299/1a90.json

```diff
@@ -0,0 +1,133 @@
+// Technisat series switches
+// Single-Switch
+{
+	"manufacturer": "TechniSat",
+	"manufacturerId": "0x0299",
+	"label": "Single-Switch",
+	"description": "Single-Switch",
```

Comment:
```suggestion
	"label": "Single-Switch 0300/9499",
	"description": "On/Off switch flush mount, BJ",
```
Does that make sense to be a bit more precise?

Reply:
That makes sense